tests/catalog/test_rest.py

```diff
@@ -701,17 +702,17 @@ def test_load_table_404(rest_mock: Mocker) -> None:
     assert "Table does not exist" in str(e.value)
 
 
-def test_table_exist_200(rest_mock: Mocker) -> None:
+def test_table_exists_200(rest_mock: Mocker) -> None:
```

Comment:
This is interesting, the integration tests aren't working?

Reply:
We don’t have integration tests for the catalog interactions, so I created an issue to implement a test suite: https://github.com/apache/iceberg-python/issues/1010

Reply:
SGTM, thanks!